plugins/scaffolder/api-report.md

```diff
@@ -32,61 +32,66 @@ import { TemplateEntityV1beta2 } from '@backstage/plugin-scaffolder-common';
 // Warning: (ae-missing-release-tag) "createScaffolderFieldExtension" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export function createScaffolderFieldExtension<T = any>(
-  options: FieldExtensionOptions<T>,
-): Extension<() => null>;
+export function createScaffolderFieldExtension<
+  TReturnValue = unknown,
+  TInputProps = unknown,
+>(
+  options: FieldExtensionOptions<TReturnValue, TInputProps>,
+): Extension<TInputProps & (() => null)>;
```

Comment:
not sure how to interpret this type 🤔  so it's something that has a set of props but it's also a function that takes _no_ args?